### PR TITLE
BIGQuery Data source ver 1.0.3

### PR DESCRIPTION
**Pull request by avivl**

### CODE REVIEW

**Comment by avivl** [comment]
@daniellee @alexanderzobnin  @schmidtl

File: repo.json [hunk 1]
```diff
@@ -3261,6 +3261,11 @@
           "version": "1.0.2",
           "commit": "d41609738582c4600c441fce3ce1667c28193937",
           "url": "https://github.com/doitintl/bigquery-grafana"
+        },
+        {
+          "version": "1.0.2",
```
[the remark this answers is not in the document]
Comment (avivl) [reply repo.json]: Thanks! good catch. I also added v 1.0.4

**Comment by avivl** [comment]
@briangann can you review, please?

**Comment by avivl** [comment]
@daniellee @alexanderzobnin @schmidtl @briangann 
Can this be merged?

**Comment by daniellee** [comment]
Published and merged. Sorry for the delay.

**Pull request merged**